applications/callflow/test/cf_ring_group_tests.erl

```diff
@@ -28,3 +28,82 @@ weighted_random_sort_test_() ->
       || X <- Endpoints
      ]
     ].
+
+%% HELP-260010392
+help_10392_test_() ->
+    {'setup'
+    ,fun setup_help_10392/0
+    ,fun cleanup_help_10392/1
+    ,fun(_ReturnOfSetup) ->
+             help_10392()
+     end
+    }.
+
+setup_help_10392() ->
+    {'ok', _} = application:ensure_all_started('kazoo_bindings'),
+    kz_fixturedb_util:start_me('true').
+
+cleanup_help_10392(Pid) ->
+    'ok' = application:stop('kazoo_bindings'),
+    kz_fixturedb_util:stop_me(Pid).
+
+help_10392() ->
+    AccountId = <<"account0000000000000000000000001">>,
+    UserId = <<"user0000000000000000000000000001">>,
+    DeviceId = <<"device00000000000000000000000001">>,
+    GroupId = <<"group000000000000000000000000001">>,
+
+    Endpoint = help_10392_user_endpoint(UserId),
+    User = kz_json:set_value(<<"delay">>, 0, Endpoint),
+    Group = help_10392_group_endpoint(GroupId),
+    EndpointDelay10 = {DeviceId
+                      ,help_10392_add_source(help_10392_add_group_weight(Endpoint))
+                      },
+    EndpointDelay0 = {DeviceId, help_10392_add_source(User)},
+
+    Call = kapps_call:set_account_id(AccountId, kapps_call:new()),
+
+    Resp0 = cf_ring_group:resolve_endpoint_ids(help_10392_data([User, Group]), Call),
+    Resp1 = cf_ring_group:resolve_endpoint_ids(help_10392_data([Group, User]), Call),
+    Resp2 = cf_ring_group:resolve_endpoint_ids(help_10392_data([User, User]), Call),
+    Resp3 = cf_ring_group:resolve_endpoint_ids(help_10392_data([Group, Group]), Call),
+
+    Expected0 = [EndpointDelay0, EndpointDelay10],
+    Expected1 = [EndpointDelay10, EndpointDelay0],
```

Comment:
I was under the impression that we were going to sort the ring group based on the delay. So this expectation is backwards. My understanding was `resolve_endpoint_ids([User, Group], Call)` and `resolve_endpoint_ids([Group, User], Call)` would result in the same listing: `[EndpointDelay0, EndpointDelay10]`.

Reply:
The sorting by delay happens at a higher level https://github.com/2600hz/kazoo/blob/a75f05de3be1a69485e42f442284e1d1df61ff65/applications/callflow/src/module/cf_ring_group.erl#L134

The functionality being tested here is just in charge of resolving the endpoints for each Ring Group member.

If you want I can add another test for testing the ordering by delay.